tests/ClientTest.php

```diff
@@ -99,7 +100,7 @@ public function testBulkCallableResolution(): void
         $stub = new CallableResponseClientStub();
         $client = $this->createInstance($stub);
 
-        $client->bulkUpdate('index', ['1' => 'document']);
+        $client->bulkUpdate([new DocumentUpdate('index', '1', 'document')]);
 
         // If call was successful there should be no return/exception
         $this->addToAssertionCount(1);
```

Comment:
We have further assertion, so I guess we can remove `addToAssertionCount`?

Reply:
This one looks more complicated because it involves dealing with futures